content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -94,3 +94,86 @@ files, etc.) can be stored in Git.
 
 > Please note that you need to `dvc push` in order to share or backup the DVC
 > cache contents.
+
+## Organization patterns
+
+While internally all experiments are special branches off a baseline (see
+[Overview](/doc/user-guide/experiment-management/experiments-overview)), it's up
+to you to decide how to organize them once completed. Here are the main
+alternatives:
+
+### Git commits, tags, and branches
+
+Use the repo's "time dimension" to distribute your experiments. This makes the
+most sense for experiments that build on each other. Git-based experiment
+structures are especially helpful along with Git history exploration tools [like
+GitHub]. Example:
+
+![](/img/exp-branches.png) _From our [example-dvc-checkpoints] repo_
+
+[example-dvc-checkpoints]:
+  https://github.com/iterative/example-dvc-checkpoints/network
+
+### Directories
+
+The project's "space dimension" can be structured with directories (folders) to
+organize experiments. Useful when you want to see all your experiments at the
+same time (without switching versions) by just exploring the file system.
+Example:
+
+```
+├── data
+│   └── labels.raw
+├── dvc.yaml
+└── experiments
+    ├── cnn_128
+    ├── cnn_64
+    └── linear
+```
+
+(ℹ️) When your `dvc.yaml` files are organized inside recursive subfolders, you
+can run their pipeline(s) using `dvc run --recursive`.
+
+> 📖 See also [Running all pipelines]
+
+### Hybrid
+
+Combining an intuitive directory structure with a good repo branching strategy
+tends to be the best option for complex projects. Completely independent
+experiments live in separate directories, while their progress can be found in
+different branches. Example:
+
+<cards>
+  <card>
+    v0.1.0
+
+    ```
+    └── experiments
+        ├── cnn_128
+        └── cnn_64
+    ```
+
+  </card>
+  <card>
+    v0.2.0
+
+    ```
+    └── experiments
+        ├── cnn_128
+        └── cnn_512
+    ```
```

Comment:
I think instead of naming these `cnn_128`, `cnn_512`, etc. it may be more understandable to use the same words and common steps in an ML pipeline, like `data augmentation`, `dataset1`, `dataset2` etc. `cnn` is always in the training phase, and may be different versions of `model` directory.